simulation_ws/src/cloudwatch_simulation/scripts/spawn_performer.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# just print this out
+ign service -s /world/default/level/set_performer --reqtype ignition.msgs.StringMsg --reptype ignition.msgs.Boolean --timeout 2000 --req 'data: "turtlebot3_waffle_pi_1"'
```

Comment:
Aren't there 10 TurtleBots? Can you do this for `turtlebot3_waffle_pi_[1-10]`? Thanks.